packs/feats/animal-swiftness.json

```diff
@@ -24,7 +24,67 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Actor.Speed.Type.Land",
+                        "value": {
+                            "increase": 10,
+                            "type": "land"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Actor.Speed.Type.Fly",
+                        "value": {
+                            "increase": 5,
+                            "type": "fly"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Actor.Speed.Type.Swim",
+                        "value": {
+                            "increase": 5,
+                            "type": "swim"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Actor.Speed.Type.Climb",
+                        "value": {
+                            "increase": 5,
+                            "type": "climb"
+                        }
+                    }
+                ],
+                "flag": "speedType",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.Speed"
+            },
+            {
+                "key": "RollOption",
+                "option": "speed-type:{item|flags.pf2e.rulesSelections.speedType.type}"
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "speed",
+                "value": "{item|flags.pf2e.rulesSelections.speedType.increase}"
+            },
+            {
+                "key": "BaseSpeed",
+                "predicate": [
+                    "change-shape:hybrid",
+                    {
+                        "or": [
+                            "speed-type:fly",
+                            "speed-type:climb",
+                            "speed-type:swim"
+                        ]
+                    }
+                ],
+                "selector": "{item|flags.pf2e.rulesSelections.speedType.type}",
+                "value": "(@actor.attributes.speed.value + 5)"
```

Comment:
Why the +5? Shouldn't `@actor.attributes.speed.value` already take the land speed bonus into account?

Reply:
It doesn't 

If you build an actor with the original feat, it only adds the base speed without the flat modifier.